src/Sylius/Bundle/ShopBundle/Resources/views/Product/Show/Tab/_attributes.html.twig

```diff
@@ -0,0 +1,14 @@
+<div class="ui bottom attached tab segment" data-tab="attributes">
+    <table id="sylius-product-attributes" class="ui definition table">
+        <tbody>
+        {% for attribute in product.attributes %}
+            <tr>
+                <td class="sylius-product-attribute-name">{{ attribute.name }}</td>
+                <td class="sylius-product-attribute-value">
+                    {% include [('SyliusAttributeBundle:Types:'~attribute.attribute.type~'.html.twig')] with {'attribute': attribute} %}
```

Comment:
How about fallback to default when no custom template defined? `include [('SyliusAttributeBundle:Types:'~attribute.attribute.type~'.html.twig'), 'SyliusAttributeBundle:Types:default.html.twig'] with ...`

This way we can get rid of `integer` and `text` templates and simply render `{{ attribute.value }}` in default template.